src/plugins/legacy.js

```diff
@@ -88,6 +89,7 @@ export function convertFromV5 (c: LegacyCommand): Class<Command<*>> {
     V5.flags.remote = Flags.remote()
   }
   if (c.needsOrg || c.wantsOrg) {
+    new Output().warn(`--org in ${c.topic}${c.command ? ':' + c.command : ''} is deprecated`)
```

Comment:
you should be able to use `this.out`. You should never create a new output because it won't reuse the existing config object if you do this